config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  get 'users/new'
+  devise_for :users
+  devise_scope :user do
```

Comment:
No, from what I understand, it is necessary to generate the "/users" routes for CRUD.  The ```devise_for``` method creates the routes for the session and password.